### PR TITLE
Add clasp project

### PR DESCRIPTION
**Pull request by bnico99**

### CODE REVIEW

File: varats/varats/projects/cpp_projects/clasp.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+)
+from varats.utils.settings import bb_cfg
+
+
+class Hypre(VProject, ReleaseProviderHook):
+    """
+    clasp is an answer set solver for (extended) normal and disjunctive logic
+    programs.
+
+    (fetched by Git)
```
Comment (vulder) [inline varats/varats/projects/cpp_projects/clasp.py]:
```suggestion
```
Old copy paste error that keeps on reappearing :rofl:

**Review by vulder — APPROVED** [review]

File: varats/varats/projects/cpp_projects/clasp.py (hunk 1 above)
Comment (boehmseb) [reply varats/varats/projects/cpp_projects/clasp.py]: Well that answers my question :smile:

**Review by boehmseb — APPROVED** [review]

File: varats/varats/projects/cpp_projects/clasp.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+)
+from varats.utils.git_util import (
+    RevisionBinaryMap,
+    ShortCommitHash,
+    FullCommitHash,
+)
+from varats.utils.settings import bb_cfg
+
+
+class Hypre(VProject, ReleaseProviderHook):
```
Comment (vulder) [inline varats/varats/projects/cpp_projects/clasp.py]:
```suggestion
class Clasp(VProject, ReleaseProviderHook):
```

File: varats/varats/projects/cpp_projects/clasp.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+            limit=None,
+            shallow=False
+        )
+    ]
+
+    @staticmethod
+    def binaries_for_revision(
+        revision: ShortCommitHash
+    ) -> tp.List[ProjectBinaryWrapper]:
+        binary_map = RevisionBinaryMap(get_local_project_git_path(Hypre.NAME))
```
Comment (vulder) [inline varats/varats/projects/cpp_projects/clasp.py]:
```suggestion
        binary_map = RevisionBinaryMap(get_local_project_git_path(Clasp.NAME))
```

File: varats-core/varats/project/project_domain.py [hunk 4]
```diff
@@ -25,6 +25,7 @@ class ProjectDomains(Enum):
     SECURITY = "Security"
     SOLVER = "Solver"
     SIGNAL_PROCESSING = "Signal processing"
+    SOLVER = "Solver"
```
Comment (vulder) [inline varats-core/varats/project/project_domain.py]:
```suggestion
```

**Pull request merged**